sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -254,6 +255,14 @@ export default {
       }
     };
 
+    const customEventCallback = async (form: any, event: any) => {
```

Comment:
we know the type of `form`, we can try to add an interface or it

Reply:
Interface added